sanic_openapi/doc.py

```diff
@@ -135,8 +135,9 @@ def __init__(self, cls, *args, object_name=None, **kwargs):
         self.cls = cls
         self.object_name = object_name or cls.__name__
 
-        if self.cls not in definitions:
-            definitions[self.cls] = (self, self.definition)
+        register_as = object_name or "{}.{}".format(cls.__module__, cls.__qualname__)
+        if register_as not in definitions:
+            definitions[register_as] = (self, self.definition)
```

Comment:
Humm..., how about this:

```python
self.object_name = object_name or "{}.{}".format(cls.__module__, cls.__qualname__)

if self.object_name not in definitions:
    definitions[self.object_name] = self.definition
```

I think this can make `definitions` more simplified.

Reply:
Well, my reasoning is this:
`object_name` is displayed on the swagger UI in multiple places (API endpoint documentations, models list). I definitely would not recommend showing the full qualified names of model classes there by default, because the documentation would leak internal implementation and project structure details.
(Aesthetics is very subjective, but i think not having full qualified class names on the UI looks better and is more readable.)